### DISCUSSION TITLE
Differences from books

### DISCUSSION THREAD

**Discussion by cloford** [Homebrew/discussions#2131] [Getting started]
I have two questions.

1,The book says install/master/, but now when I check it, it says install/HEAD. What is the difference between these?

/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"

/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"

2, I would like to know about the following two commands, are they necessary to install JDK and maven using homebrew?

brew install --cask java
brew install maven

Sorry for the rudimentary question, but please help me.

**Comment by SMillerDev** [comment]
I'm not sure what book you're referring to. But the install command on <https://brew.sh> is the correct command.
If you want to install maven, just `brew install maven` is enough.

**Reply by cloford** [reply]
>>However, the install command at https://brew.sh is the correct command.

/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
Are you talking about this one?
